app/main/views/g_cloud.py

```diff
@@ -609,6 +609,38 @@ def why_didnt_you_award_contract(framework_framework, project_id):
     abort(404)
 
 
+@direct_award.route(
+    '/<string:framework_framework>/projects/<int:project_id>/tell-us-about-contract',
+    methods=['GET', 'POST']
+)
+def tell_us_about_contract(framework_framework, project_id):
+    all_frameworks = data_api_client.find_frameworks().get('frameworks')
+    framework = framework_helpers.get_latest_live_framework(all_frameworks, framework_framework)
+
+    # Get the requested Direct Award Project.
+    project = data_api_client.get_direct_award_project(project_id=project_id)['project']
+
+    form = TellUsAboutContractForm()
+
+    if form.validate_on_submit():
+        flash(Markup(f"""You've updated '{project['name']}'"""), 'success')
+        return redirect(url_for('.view_project', framework_framework=framework_framework, project_id=project_id))
+
+    errors = [
+        {
+            "input_name": input_name,
+            "question": getattr(form, input_name).label
+        } for input_name in form.errors.keys()]
```

Comment:
Building the list of errors in this way might be worth abstracting out into a separate function?

Reply:
Definitely

Reply:
Looks like @samuelhwilliams beat me to this 👍 https://github.com/alphagov/digitalmarketplace-buyer-frontend/pull/763#discussion_r191420911

Reply:
If we're in a position to bring in that version of dm-utils then go for it, but if not then this can wait as long as there's a plan to do that.

Reply:
Yes please - let's pull in the error formatter:

```
from dmutils.forms import get_errors_from_wtform

errors = get_errors_from_wtform(form)
```

Reply:
@samuelhwilliams is working on getting this done, apparently updating to `dmutils` with the helpers is not a straightforward thing 😢